tests/kafkatest/services/copycat.py

```diff
@@ -77,7 +78,52 @@ def clean_node(self, node):
         node.account.ssh("rm -rf /mnt/copycat.pid /mnt/copycat.log /mnt/copycat.properties  " + " ".join(self.config_filenames() + self.files), allow_fail=False)
 
     def config_filenames(self):
-        return ["/mnt/copycat-connector-" + str(idx) + ".properties" for idx, template in enumerate(self.connector_config_templates)]
+        return ["/mnt/copycat-connector-" + str(idx) + ".properties" for idx, template in enumerate(self.connector_config_templates or [])]
+
+
+    def list_connectors(self, node=None):
+        return self._rest('/connectors', node=node)
+
+    def create_connector(self, config, node=None):
+        create_request = {
+            'name': config['name'],
+            'config': config
+        }
+        return self._rest('/connectors', create_request, node=node, method="POST")
+
+    def get_connector(self, name, node=None):
+        return self._rest('/connectors/' + name, node=node)
+
+    def get_connector_config(self, name, node=None):
+        return self._rest('/connectors/' + name + '/config', node=node)
+
+    def set_connector_config(self, name, config, node=None):
+        return self._rest('/connectors/' + name + '/config', config, node=node, method="PUT")
+
+    def get_connector_tasks(self, name, node=None):
+        return self._rest('/connectors/' + name + '/tasks', node=node)
+
+    def delete_connector(self, name, node=None):
+        return self._rest('/connectors/' + name, node=node, method="DELETE")
+
+    def _rest(self, path, body=None, node=None, method="GET"):
+        if node is None:
+            node = random.choice(self.nodes)
+
+        meth = getattr(requests, method.lower())
+        url = self._base_url(node) + path
+        resp = meth(url, json=body)
```

Comment:
Since our tests will fail without it, I suggest adding enable_dns=true to vagrant/system-test-Vagrantfile.local